install-deps.sh

```diff
@@ -35,7 +35,7 @@ function install_seastar_deps {
 function munge_ceph_spec_in {
     local OUTFILE=$1
     sed -e 's/@//g' -e 's/%bcond_with make_check/%bcond_without make_check/g' < ceph.spec.in > $OUTFILE
-    if type python2 > /dev/null 2>&1 ; then
+    if [ ! $force_python3 ] && type python2 > /dev/null 2>&1 ; then
```

Comment:
#25181 also tries address this issue. and i think it's better as it's simpler.

Reply:
Agreed -- I like that one better as well

Reply:
(commit dropped)